### DISCUSSION TITLE
WebdriverIO V7 Multiremote: Error: Object literal may only specify known properties, and 'admin' does not exist in type 'DesiredCapabilities | W3CCapabilities'.ts(2353)

### DISCUSSION THREAD

**Discussion by AliUzairMehdi** [webdriverio/webdriverio#13886] [Q&A]
I have been trying to use the multiremote feature and i'm unable to because of this error:
`Object literal may only specify known properties, and 'admin' does not exist in type 'DesiredCapabilities | W3CCapabilities'.ts(2353)`

my configuration file has the following capabilities:

 capabilities: [{
    admin: {
      capabilities: {
        browserName: `${browserToRun}`
      }
    },
    storefront: {
      capabilities: {
        browserName: `${browserToRun}`
      }
    }
  }],

**Comment by damencho** [comment]
Try to use:
```
capabilities: {
  admin: {
    capabilities: {
      browserName: ${browserToRun}
    }
  },
  storefront: {
    capabilities: {
      browserName: ${browserToRun}
    }
  }
},
```

**Comment by christian-bromann** [comment]
Setting multiremote capabilities like this isn't supported in WebdriverIO v7. I recommend to update to v9.

**Reply by AliUzairMehdi** [reply]
@christian-bromann i have tried to create a small boilerplate just to make sure things are working.
I have used the same capabilities:

```
capabilities: {
  admin: {
    capabilities: {
      browserName: ${browserToRun}
    }
  },
  storefront: {
    capabilities: {
      browserName: ${browserToRun}
    }
  }
},

```
Object literal may only specify known properties, and 'admin' does not exist in type 'RequestedStandaloneCapabilities'.

And sorry if the problem is too obvious and i am unable to see it. I'm just learning right now and new to this :)

**Reply by christian-bromann** [reply]
Can you provide a minimal reproducible example?
